static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -64,7 +92,8 @@ function BreadcrumbItem({
   startTimestampMs,
   style,
 }: Props) {
-  const {color, description, projectSlug, title, type} = getDetails(crumb);
+  const {color, description, projectSlug, title, type, timestampMs} =
+    getCrumbOrFrameData(crumb);
```

Comment:
this `getCrumbOrFrameData` doesn't need to stick around forever.

But when i'm using `frame.tsx` it's kind of annoying to have 4 different functions (8 switch statements) to keep in sync, so i might refactor that to be one method that returns everything (color, title, desc, icon) as a single object. I think only `getColor` is called individually, all others are needed at the same time.